test/util/translatePlatformKey.js

```diff
@@ -0,0 +1,26 @@
+const { Key } = require('selenium-webdriver');
+const isMacOS = require('./isMacOS');
+
+const MAC_KEY_MAPPINGS = {
+  [Key.CONTROL]: Key.META,
+};
+
+/**
+ * Translates a key or key combination for the current OS
+ *
+ * @param {string|string[]} keys - The key(s) to translate
+ * @returns {string[]} - The translated key(s) as a flat array ready for spreading
+ */
+function translatePlatformKey(keys) {
```

Comment:
This function is potentially prematurely complex. I wanted to support instances when a platform key translation may be represented by chord -> single key or single key -> chord. We do not need this functionality now since we don't use any examples of this. I am open to reverting to a map if that is preferred.

Reply:
It potentially is but I could see the need for this as we move to ensuring tests are platform agnostic. Perhaps providing a clear example in the JSDoc will also be helpful.